boards/arm/disco_l475_iot1/disco_l475_iot1.dts

```diff
@@ -116,6 +117,18 @@
 		spi-max-frequency = <2000000>;
 		label = "SPBTLE-RF";
 	};
+
+	wifi0: ism43362@0 {
```

Comment:
It should be ism43362@1 to match the reg property.

Reply:
indeed!